src/condition_codes.rs

```diff
@@ -60,4 +60,33 @@ impl ConditionCodes {
     pub fn reset_aux_carry(&mut self) {
         self.aux_carry = false
     }
+
+    pub fn flags_to_psw(&self) -> u8 {
+        let mut psw: u8 = 0x2;
+        if self.sign {
+            psw += 0x80
+        };
+        if self.carry {
+            psw += 0x1
+        };
+        if self.zero {
+            psw += 0x40
+        };
+        if self.parity {
+            psw += 0x4
```

Comment:
I updated these to remove the prepended 0 to follow the conventions we began previously.